sources/packages/api/src/route-controllers/education-program/models/save-education-program.dto.ts

```diff
@@ -8,9 +8,44 @@ import {
   getISODateOnlyString,
   getUserFullName,
 } from "../../../utilities";
+
 /**
- * Dto that represents education programs form object.
+ * Dto that represents education program with offering form object.
  */
+export class EducationProgramWithOfferingsAPIOutDTO {
```

Comment:
These DTOs seem to be also in the wrong file. We can take care of that during the EducationProgram refactor.